### DISCUSSION TITLE
好奇ing

### DISCUSSION THREAD

**Discussion by gaoxuerong** [thinkjs/thinkjs#1673] [General]
thinkjs 是停更了嘛

**Comment by zhangbowy** [comment]
据说是因为比较完善了!

**Comment by zhangbowy** [comment]
你需要什么更新

**Comment by zhangbowy** [comment]
好久没发版本 对开发者而言就是缺乏安全感？？哈哈哈

**Comment by zhangbowy** [comment]
![image](https://user-images.githubusercontent.com/33514252/123057110-2c382200-d43a-11eb-884e-2fe885801336.png)

**Comment by lizheming** [comment]
目前大的需求人力原因可能不会有，不过日常的维护还是有在的，使用上有任何问题可以 issue 反馈，我都会尽快跟进的。也欢迎感兴趣的同学进行 Pull Request。
